tests/xtro-sharpie/tvOS-UIKit.ignore

```diff
@@ -63,4 +63,8 @@
 ## fixed for XAMCORE_4_0
 !incorrect-protocol-member! UIFocusItem::frame is REQUIRED and should be abstract
 !incorrect-protocol-member! UIFocusEnvironment::focusItemContainer is REQUIRED and should be abstract
-!incorrect-protocol-member! UIFocusEnvironment::parentFocusEnvironment is REQUIRED and should be abstract
\ No newline at end of file
+!incorrect-protocol-member! UIFocusEnvironment::parentFocusEnvironment is REQUIRED and should be abstract
+
+## not supported as per introspection, rdar filled: 42851110
+!missing-protocol-conformance! NSAttributedString should conform to NSItemProviderReading (defined in 'NSAttributedString_ItemProvider' category)
+!missing-protocol-conformance! NSAttributedString should conform to NSItemProviderWriting (defined in 'NSAttributedString_ItemProvider' category)
```

Comment:
For next time: add a newline at the end (makes the diff for future additions simpler).